.github/workflows/test.yml

```diff
@@ -26,28 +26,44 @@ jobs:
         run: find ./ -type f -name "*-test.sh" -exec "./{}" \;
 
   integration-test:
-    runs-on: ubuntu-20.04
+    runs-on: ${{ matrix.os }}
     name: "integration test"
     strategy:
       fail-fast: false
       matrix:
         include:
           # Disabled due to https://github.com/getsentry/self-hosted/issues/1415
-          # - compose_version: "1.28.0"
+          # - os: "ubuntu-20.04"
+          #   compose_version: "1.28.0"
           #   compose_path: "/usr/local/bin"
-          # - compose_version: "1.29.2"
+          # - os: "ubuntu-20.04"
+          #   compose_version: "1.29.2"
           #   compose_path: "/usr/local/bin"
-          - compose_version: "v2.0.1"
+          - os: "ubuntu-20.04"
+            compose_version: "v2.0.1"
+            compose_path: "/usr/local/lib/docker/cli-plugins"
+          - os: "ubuntu-20.04"
+            compose_version: "v2.2.3"
             compose_path: "/usr/local/lib/docker/cli-plugins"
-          - compose_version: "v2.2.3"
+          - os: "macos-11"
+            compose_version: "v2.2.3"
             compose_path: "/usr/local/lib/docker/cli-plugins"
     env:
       COMPOSE_PROJECT_NAME: self-hosted-${{ strategy.job-index }}
     steps:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - name: Setup docker on macOS
+        if: ${{ startsWith(matrix.os, 'macos-') }}
+        uses: docker-practice/actions-setup-docker@v1
+
+      - name: Install coreutils on macOS
```

Comment:
I don't think this should be needed. I don't have coreutils installed on my laptop and I can set up Sentry fine. Was there an error that you ran into that led you to add this?

Reply:
This used to be needed before https://github.com/getsentry/self-hosted/issues/941.

Reply:
CI failed before adding `coreutils`, here is the log because of `timeout` command here.
https://github.com/getsentry/self-hosted/runs/7371688591?check_suite_focus=true#step:5:1612

Anyway to be certain I will revert the commit adding `coreutils` after resolving conflicts and rebasing and then we can take a better look.